cassandra-adapter/src/main/java/org/polypheny/db/adapter/cassandra/CassandraPhysicalNameProvider.java

```diff
@@ -18,27 +18,185 @@
 
 
 import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
 import org.polypheny.db.catalog.Catalog;
+import org.polypheny.db.catalog.entity.CatalogColumn;
+import org.polypheny.db.catalog.entity.CatalogColumnPlacement;
 import org.polypheny.db.catalog.entity.CatalogTable;
 import org.polypheny.db.catalog.exceptions.GenericCatalogException;
+import org.polypheny.db.catalog.exceptions.UnknownColumnException;
 import org.polypheny.db.catalog.exceptions.UnknownTableException;
 
 
 public class CassandraPhysicalNameProvider {
 
+    private final static Pattern idRevPattern = Pattern.compile( "^(col|tab|sch)([0-9]+)(?>r([0-9]+))?$" );
+
     private final Catalog catalog;
+    private final int storeId;
 
+    private final String DEFAULT_SCHEMA = "public";
 
-    public CassandraPhysicalNameProvider( Catalog catalog ) {
+    public CassandraPhysicalNameProvider( Catalog catalog, int storeId ) {
         this.catalog = catalog;
+        this.storeId = storeId;
+    }
+
+
+    public String generatePhysicalColumnName( long columnId ) {
+        return "col" + columnId;
+    }
+
+
+    public String generatePhysicalTableName( long tableId ) {
+        return "tab" + tableId;
+    }
+
+
+    public String generatePhysicalSchemaName( int schemaId ) {
+        // TODO JS: implement cassandra schemas
+        return "cassandra";
+//        return "sch" + schemaId;
+    }
+
+
+    public String getPhysicalSchemaName( int schemaId ) {
+        return generatePhysicalSchemaName( schemaId );
+    }
```

Comment:
Why not returning `DEFAULT_SCHEMA` here?

Reply:
`DEFAULT_SCHEMA` refers to the default logical schema name, not the physical schema name. The constant should probably reflect that.